jest.config.js

```diff
@@ -23,10 +23,13 @@ module.exports = {
     "packages/remix-server-runtime",
     "packages/remix-testing",
   ],
+  reporters:
+    process.env.GITHUB_ACTIONS === null
```

Comment:
should be `==` - env variables that aren't set are `undefined`, not `null`

Reply:
Fixed in 517e40bae90ddec7a94a93f06c206cf93363210f